rust/bindings_napi/Cargo.toml

```diff
@@ -15,5 +15,11 @@ napi-derive = "2.13.0"
 parse_ast = { path = "../parse_ast" }
 xxhash = { path = "../xxhash" }
 
+[target.'cfg(not(target_os = "linux"))'.dependencies]
+mimalloc-rust = { version = "0.2" }
+
+[target.'cfg(all(target_os = "linux", not(all(target_env = "musl", target_arch = "aarch64"))))'.dependencies]
```

Comment:
@sapphi-red: the linux arm64 arch exclusion is intentional?

Reply:
Yes, it is. The examples I shown in the OP all excludes linux+musl+arm64. I don't know the reason, but as I cannot easily check the actual behavior, I went with a safer way.

Reply:
That issue https://github.com/microsoft/mimalloc/issues/556 seems to say that since version 2.0.5, it works on linux arm64. So if the mimalloc-rust bindings is against that version or superior, it should also work.

Reply:
The rust bindings use 1.7.9: https://github.com/LemonHX/mimalloc-rust#:~:text=mimalloc%201.7.9%20stable
So that seems to be the reason.